generate.pl

```diff
@@ -195,13 +195,14 @@ =head1 DESCRIPTION
 =cut
 
 __DATA__
-FROM buildpack-deps:{{_tag}}
+FROM buildpack-deps:{{_tag}} as build
 LABEL maintainer="Peter Martini <PeterCMartini@GMail.com>, Zak B. Elep <zakame@cpan.org>"
 
 COPY *.patch /usr/src/perl/
 WORKDIR /usr/src/perl
 
-RUN curl -SL {{url}} -o perl-{{version}}.tar.{{type}} \
+RUN apt-get update && apt-get install --no-install-recommends less \
```

Comment:
Why is `less` installed here?

Reply:
I saw on a docker build for 5.18 that is was complaining that less wasn't found. So I added it.

Reply:
Right, just tried a rebuild here:

```
...
perl-5.18.4.tar.bz2: OK
patching file ext/Errno/Errno_pm.PL
patching file hints/linux.sh
First let's make sure your kit is complete.  Checking...
Locating common programs...
./Configure: 2469: ./Configure: less: not found
Checking compatibility between /bin/echo and builtin echo (if any)...
Symbolic links are supported.
Checking how to test for symbolic links...
You can test for symbolic links with 'test -h'.
...
```

It doesn't seem essential to the build though (it still gets us a working perl, and image users can just install `less` later) so it can probably be left off.

Reply:
I removed it. But I saw a whole lot of activity in issue #52, so I guess this PR will not be merged.

Reply:
No problem, good to see things moving :+1: